src/Symfony/Component/Filesystem/Tests/FilesystemTest.php

```diff
@@ -842,6 +842,16 @@ public function providePathsForMakePathRelative()
             array('/a/aab/bb/', '/b/aab', '../../a/aab/bb/'),
             array('/aab/bb', '/aa', '../aab/bb/'),
             array('/aab', '/aa', '../aab/'),
+            array('/aa/bb/cc', '/aa/dd/..', 'bb/cc/'),
+            array('/aa/../bb/cc', '/aa/dd/..', '../bb/cc/'),
+            array('/aa/bb/../../cc', '/aa/../dd/..', 'cc/'),
+            array('../aa/bb/cc', '/aa/dd/..', 'bb/cc/'),
+            array('../../aa/../bb/cc', '/aa/dd/..', '../bb/cc/'),
```

Comment:
These two are not absolute paths while makePathRelative() says it accepts two absolute paths.